frontend/src/main/web/lib/constants/Views.js

```diff
@@ -0,0 +1,74 @@
+import React from 'react';
+import Router from 'react-router';
+import StringUtils from '../utils/StringUtils';
+import RootContent from '../components/RootContent';
+import UserProfile from '../components/UserProfile';
+import SystemGlossary from '../components/SystemGlossary';
+
+var Views = {
```

Comment:
Change to `views` (only constructors and React components usually begin with caps).